tutorial/basestate/README.md

```diff
@@ -0,0 +1,91 @@
+2.66742102105863349715e-19 -2.2382279707625282588e-19 0.0499991526205256775484 4.46604035770848181858e-18 5.33775604729435379127e-18 -1.92613521797670789656e-21 9.86001750322409632563e-19 -8.8071622250169726935e-19 8.53451684918425657056e-06 1.76120401511952139491e-17 1.9743566732536308891e-17 -2.53183071336154027179e-21 1.30212116084367548068e-16 -3.72388682882364640763e-16 -8.62087312274439909743e-05 7.40595074127527750534e-15 2.66396774171890798356e-15 4.71353545744632500157e-19
+2.66742102105863349715e-19 -2.2382279707625282588e-19 0.0499991526205256775484 4.46604035770848181858e-18 5.33775604729435379127e-18 -1.92613521797670789656e-21 9.86001750322409632563e-19 -8.8071622250169726935e-19 8.53451684918425657056e-06 1.76120401511952139491e-17 1.9743566732536308891e-17 -2.53183071336154027179e-21 1.30212116084367548068e-16 -3.72388682882364640763e-16 -8.62087312274439909743e-05 7.40595074127527750534e-15 2.66396774171890798356e-15 4.71353545744632500157e-19
+2.66742102105863349715e-19 -2.2382279707625282588e-19 0.0499991526205256775484 4.46604035770848181858e-18 5.33775604729435379127e-18 -1.92613521797670789656e-21 9.86001750322409632563e-19 -8.8071622250169726935e-19 8.53451684918425657056e-06 1.76120401511952139491e-17 1.9743566732536308891e-17 -2.53183071336154027179e-21 1.30212116084367548068e-16 -3.72388682882364640763e-16 -8.62087312274439909743e-05 7.40595074127527750534e-15 2.66396774171890798356e-15 4.71353545744632500157e-19
+
+~~~
+
+On each line are listed 18 elements:
+
+- X, Y, Z coordinates of the origin of the base link frame expressed in the world reference frame;
+- Roll, Pitch and Yaw angles expressing the orientation of the base link frame with respect to the world reference frame, in radians;
+- Linear velocity of the origin of the base link frame expressed in the world reference frame;
+- Angular velocity of the origin of the base link frame expressed in the world reference frame;
+- Linear acceleration of the origin of the base link frame expressed in the world reference frame;
+- Angular acceleration of the origin of the base link frame expressed in the world reference frame;
+
+As you can observe from the basestate plugin output, the base link frame starts at 0.2m height on Z direction (the 3rd value printed on each line) and when the simulation runs the cube falls on the ground; the Z value then remains constant to ~0.05m.
+
+> [!NOTE]
+> If you are using Linux on WSLg and you have the following error:
+>
+> ~~~
+> OGRE EXCEPTION(9:UnimplementedException):  in GL3PlusTextureGpu::copyTo at ./RenderSystems/GL3Plus/src/OgreGL3PlusTextureGpu.cpp (line 685)
+> ~~~
+>
+> try forcing software rendering:
+>
+> ~~~
+> export LIBGL_ALWAYS_SOFTWARE=1
+> ~~~
```

Comment:
```suggestion
```

Not necessary anymore, see https://github.com/robotology/gz-sim-yarp-plugins/issues/40#issuecomment-1862555770 .

Reply:
Thanks! Should we remove this note from all the tutorials then?

Reply:
Sure, but better not do to it here, just use another PR. We can track this in https://github.com/robotology/gz-sim-yarp-plugins/issues/40#issuecomment-1862555770 .